example/example.html

```diff
@@ -4,35 +4,60 @@
     <meta charset="UTF-8">
     <title>Example</title>
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Roboto:300,500,700" rel="stylesheet" type="text/css">
-    <link rel="stylesheet" href="../build/build.css"/>
+    <link rel="stylesheet" href="../index.css"/>
+    <link rel="stylesheet" href="../node_modules/@nib-styles/v2-icons/dist/compiled.css"/>
 </head>
 <body class="v2-copy">
 
-    <div class="v2-alert v2-alert--error">
-        <div class="v2-alert__title">Error!</div>
-        Uh oh! You failed at life.
-    </div>
+  <div class="alert alert--error">
+    <div class="alert-msg--single">Alert message without title 18pt light Roboto</div>
+  </div>
 
-    <br/>
+  <p/>
 
-    <div class="v2-alert v2-alert--success">
-        <div class="v2-alert__title">Success!</div>
-        Yay! You succeeded at life.
-    </div>
+  <div class="alert alert--error">
+    <div class="alert--title">We're sorry, due to too many invalid login attempts your account is locked</div>
+    <div class="alert--message">You will be able to try again in 00:30 minutes.</div>
+  </div>
 
-    <br/>
+  <p>
 
-    <div class="v2-alert v2-alert--warning">
-        <div class="v2-alert__title">Warning!</div>
-        Warning Will Robinson! This might make your life sucky.
-    </div>
+  <div class="alert alert--success">
+    <div class="alert-msg--single">Alert message without title 18pt light Roboto</div>
```

Comment:
This looks like it has very similar styles to the title? Could it just use the title styles?